source/testnet/see-also/release-notes.rst

```diff
@@ -45,13 +74,13 @@ October 6th, 2020.
 -  Chain visualization: The connection of blocks has been made more
    stable to ensure that it progresses smoothly.
 -  `iOS Concordium ID`_ app available.
--  `Added import to app`_. It is now possible to import a file that has
+-  Added import to app. It is now possible to import a file that has
    previously been exported. This enables moving identities and accounts
    to other mobile devices and restoring from backup.
 -  µGTU. The smallest unit has been changed from 10-4 to 10-6.
 -  Bulletproofs. The core blockchain has been updated to support use of
    bulletproofs.
--  `Encrypted(shielded) amounts and transfers`_. Support for shielded
+-  Encrypted(shielded) amounts and transfers. Support for shielded
```

Comment:
Why was the link removed?

Reply:
It was linking to something that is removed in another PR. I will add a link to the new page, replacing the old one.

Reply:
Link to the newly added section on shield balances and transfers added.